setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read().replace('.. :changelog:', '')
+
+requirements = [
+    # TODO: put package requirements here
+]
+
+test_requirements = [
+    # TODO: put package test requirements here
+]
+
+setup(
+    name='panamadv',
+    version='0.1.1',
+    description="Program to calculate the DV for the given RUC for Panama.",
+    long_description=readme + '\n\n' + history,
+    author="Jose Vasconcellos",
+    author_email='info@vauxoo.com',
```

Comment:
No problem, now fixed, if you dont have any other concern merge when you consider. 
thanks for your great contribution!